contrib/fuzz/oss_fuzz_build.sh

```diff
@@ -38,7 +41,10 @@ compile_fuzzers() {
 }
 
 # This is from https://github.com/AdamKorcz/instrumentation
+git clone --depth=1 "$INSTRUMENTATION_URL"
```

Comment:
```suggestion
git clone "$INSTRUMENTATION_URL"
```

Keeping `--depth=1` will break once there are new commits at the tip of the HEAD branch.

Reply:
Ahh, yeah. Missed it. 

Thanks.